### PR TITLE
Clarify the scope of project governance

### PR DESCRIPTION
**Pull request by negz**
### Description of your changes

Following on from https://github.com/crossplane/crossplane/pull/3045 we'd like to be explicit that governance covers the crossplane-contrib organisation too, despite that organisation being a home for Crossplane extensions that fall outside our charter.

Note that I wrapped the document at 80 chars in a separate commit, so it will probably be easier to review just the non-mechanical commit.

I have:

- [x] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

**Comment by luebken** [comment]
Thanks, @negz, for starting this. TBH I'm still trying to wrap my head around this. 

> Both GitHub organizations fall under Crossplane's governance policies, but only the crossplane organization is constrained by its charter. The crossplane-contrib organization exists to provide a neutral home for Crossplane extensions. Such extensions fall outside of the project's charter and are not required to fall under the project's governance, but may optionally do so.

In the first sentence you right "Both GitHub organizations fall under Crossplane's governance policies" and in the last you write "[extensions] are not required to fall under the project's governance". 🤔 

Maybe it's helpfull to start with the intention. From my understanding we (as in the Crossplane community) want to provide https://github.com/crossplane-contrib as a vendor neutral home for the community to collaborate on Crossplane projects which are outside of the charter.

First question: Do we want to limit crossplane-contrib to extensions?

File: GOVERNANCE.md [hunk 1]
```diff
@@ -1,266 +1,313 @@
...
+developed across two GitHub organizations:
+
+* https://github.com/crossplane
+* https://github.com/crossplane-contrib
+
+Both GitHub organizations fall under Crossplane's governance policies, but only
+the crossplane organization is constrained by its [charter](CHARTER.md). The
+crossplane-contrib organization exists to provide a neutral home for Crossplane
+extensions. Such extensions fall outside of the project's charter and are not
+required to fall under the project's governance, but may optionally do so.
```
[the remark this answers is not in the document]
Comment (negz) [reply GOVERNANCE.md]: I'm not 100% sure I understand you, but I think you're saying that governance should apply to every repo in crossplane and crossplane-contrib. That's the intent. I pushed a small phrasing tweak to try make it clearer. LMK if I misunderstood.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -1,266 +1,313 @@
...
-  multiple repositories
-* Participate in the conflict resolution and voting process at the organization scope when necessary
-* Approving new repositories, or archiving repositories in the Crossplane organization
+* Review and approve central/core architecture and design changes that have
+  broad impact across multiple repositories
+* Participate in the conflict resolution and voting process at the organization
+  scope when necessary
+* Approving new repositories, or archiving repositories in the Crossplane
+  organization
 * Add and remove members in the Crossplane organization
```
[the remark this answers is not in the document]
Comment (negz) [reply GOVERNANCE.md]: I don't think we need a separate process. Let's say ask the steering committee for now.

**Comment by negz** [comment]
> In the first sentence you right "Both GitHub organizations fall under Crossplane's governance policies" and in the last you write "[extensions] are not required to fall under the project's governance".

@luebken My intention was to communicate that extensions don't need to live in contrib. Everything in contrib is under governance. I've pushed a commit to clarify.

> Do we want to limit crossplane-contrib to extensions?

No, but I'm inclined to leave the phrasing as is for now given contrib is 95%+ extensions (providers). We can always revisit if/when we have something that is not an extension that fits better in crossplane-contrib than crossplane.

**Review by jbw976 — APPROVED** [review]
Thanks for making this clarification @negz!

**Pull request merged**